Formula/gnss-sdr.rb

```diff
@@ -0,0 +1,31 @@
+class GnssSdr < Formula
+  desc "Open-source software-defined GNSS receiver"
+  homepage "https://gnss-sdr.org/"
+  url "https://github.com/gnss-sdr/gnss-sdr/archive/refs/tags/v0.0.14.tar.gz"
+  sha256 "b1636455ce70be9cb93793ee5a68faa47d8bcdafeddbcab55669f545e6271b6d"
+  license "GPL-3.0-only"
+  head "https://github.com/gnss-sdr/gnss-sdr.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "gnuradio"
+  depends_on "hdf5"
+  depends_on "openblas"
+
+  def install
+    inreplace "src/algorithms/libs/volk_gnsssdr_module/volk_gnsssdr/CMakeLists.txt" do |s|
+      s.gsub!(/^set\(prefix .+\)$/, "set(prefix \"#{prefix}\")")
+      s.gsub!(/^set\(libdir .+\)$/, "set(libdir \"#{lib}\")")
+      s.gsub!(/^set\(includedir .+\)$/, "set(include \"#{include}}\")")
+    end
+    system "cmake", "-S", ".", "-B", "build", "-DBLA_VENDOR=OpenBLAS", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    system "gnss-sdr", "-version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula